about.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+        <link rel="stylesheet" href="./style/about.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;700&family=Lato:wght@300&family=Montserrat:wght@900&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+        <title>About Conference world!</title>
+      </head>
+<body>
+ <div class="container-fluid mobile p-0 pb-5">
+    <header class="header">
+      <nav>
+        <div class="container-fluid">
+          <button class="hamburger" type="button">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </button>
+        </div>
+        <div class="container-fluid full-screen-nav full-nav">
+          <a class="navbar-brand" href="./index.html">
+            <img src="./img/logo-nav.png" alt=""  class="d-inline-block align-text-top nav-logo">
+          </a>
+          <div class="links">
+              <ul>
+                  <li class="nav-links">
+                      <a href="./index.html">Home</a>
+                  </li>
+                  <li class="nav-links">
+                      <a href="#">Program</a>
+                  </li>
+                  <li class="nav-links">
+                      <a href="#">Join</a>
+                  </li>
+                  <li class="nav-links">
+                      <a href="#">Sponsor</a>
+                  </li>
+              </ul>
+          </div>
+        </div>
+      </nav>
+      </header>
```

Comment:
Kindly add the black nav that exists in the Behance design in the `home` and `about` pages in the desktop version.
![image](https://user-images.githubusercontent.com/24703171/163293735-d3bebfcc-a0ba-4cf1-b60e-213cf9cd732d.png)